api/CustomStateSet.json

```diff
@@ -11,14 +11,7 @@
           "chrome_android": "mirror",
```

Comment:
This is all correct according to the Bugzilla bug. Related work tracked in https://github.com/mdn/content/issues/33018